pkg/state/appender.go

```diff
@@ -734,7 +734,7 @@ func (a *txAppender) appendBlock(params *appendBlockParams) error {
 			rideV6Activated:                  rideV6Activated,
 			consensusImprovementsActivated:   consensusImprovementsActivated,
 			blockRewardDistributionActivated: blockRewardDistributionActivated,
-			invokeExpressionActivated:        invokeExpressionActivated,
+			txStateSnapshotActivated:         lightNodeActivated,
```

Comment:
Maybe rename `appendTxParams.txStateSnapshotActivated` to `appendTxParams.lightNodeActivated`?